pages/a/ms-azure-guest-post.html

```diff
@@ -0,0 +1,89 @@
+title: Crate guest post for Azure - Setting up Azure for Crate
+author: Chris Ward
+description: tbc
+created: 2016-02-04
+status: publish
+post_type: post
+tags: a, b
+category: developernews
+
+# Crate guest post for Azure - Setting up Azure for Crate
+
+Crate.io has been striving to create the most scalable database in our industry. We aim to make scaling your data from millions to billions of records and beyond, as simple as possible.
+
+We have clients who write and query gigabytes of data a minute, store terabytes a day and love the performance and stability that Crate offers. But the question always remained at the back of our minds, when taken to the extreme, what is Crate really capable of?
+
+Our aims were to:
+
+- See how Crate would distribute shards and partitions across such a large cluster
+- Examine the overhead added by a the cluster management
+- See what speeds the cluster could write data at
+- Another
+- And another
+
+Microsoft Azure were generously lent us the use of 8000 cores as the infrastructure for this test and engineer time to guide us through problems and questions we might have.
+
+Microsoft hoped to prove that their Azure platform was reliable enough for a project of such rapid scaling and resource intensive.
```

Comment:
"hoped to prove" sounds a bit weak. they wanted to show the capabilities and reliability of their platform. maybe jodok can elaborate a bit more on that.